GitHubAction/GitHubAction/Presenters/Impl/GitLabOutputPresenter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace GitHubAction.Presenters.Impl;
+
+public class GitLabOutputPresenter : IOutputPresenter
+{
+    private readonly ILogger _logger;
+    public GitLabOutputPresenter(ILogger logger)
+    {
+        _logger = logger;
+    }
+
+
+    public void PresentOutputVariable(string name, string value)
+    {
+        //TODO(LLS): Figure out how output variables work in GitLab
+        _logger.LogInformation("{name}: {value}", name, value);
+    }
+
+    public void PresentInvalidArguments()
+    {
+        _logger.LogError("There was a problem with the provided arguments...");
+        Environment.Exit(400); // Bad Request
```

Comment:
Should not exit with 400 (max is 255).
Will be the same in the other presenters...